hasher-matcher-actioner/hmalib/models.py

```diff
@@ -108,6 +193,14 @@ def to_sqs_message(self) -> dict:
             "key": self.content_id,
         }
 
+    @classmethod
+    def is_total_countable(cls):
+        return True
+
+    @classmethod
+    def get_total_count_keyname(cls):
+        return "PipelinePDQHashRecord"
```

Comment:
>Rather than inferring the class name, requires that the class explicitly provide a name so that future refactoring does not reset the counters.

I have mixed feelings about this. It seems super easy to miss and I don't love the name of some of these classes so I think we will want to change it eventually. Specific counts are also something I think would be easier to migrate if needed because it is only one command per record type.  
Thoughts on adding an assert or check to log a warning if the inferred class no longer matches this hard coded value as a middle ground for now? (that way the counts wont reset but the dev will see an error/warning?)

Reply:
If it were to be missed completely, the count operations would throw a `NotImplementedError`.

If the class name is changed, the counts continue to work. Unless the dev changes the return value of this method. They should only change that if they have handwritten records to the dynamodb. I think this is where I would want to draw the line.

Reply:
Okay, I would say let's not have it match the class name from the start at all then if that is the desired outcome (to avoid confusion/a naive find+replace refactor) but this has definitely become more a nit imo.